Satori/Satori.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+      <LangVersion>8</LangVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="System.Net.Http" Version="4.3.4" />
+      <PackageReference Include="NakamaClient" Version="3.3.0" />
```

Comment:
Let's remove this as an external dependency and instead copy over the few files that would otherwise be "shared machinery" between the two DLLs. This way developers can pull in Satori in a decoupled way and optionally remove the Nakama.dll from their Unity package if they prefer after import.